src/services/jwt.service.ts

```diff
@@ -1,44 +1,46 @@
 import jwt from 'jsonwebtoken';
 import { User } from '@entities/user.entity';
 import { JWT_SECRET, ACCESS_TOKEN_LIFE } from '@config';
+import { Service } from 'typedi';
 
-export const createJWT = (user: User): Promise<string> => {
-  return new Promise((resolve, reject) => {
-    try {
-      const token = jwt.sign(
-        { data: { userId: user.id, email: user.email } },
-        JWT_SECRET || '',
-        { expiresIn: ACCESS_TOKEN_LIFE || '6h' }
-      );
-      resolve(token);
-    } catch (error) {
-      reject(new Error('Error creating JWT'));
-    }
-  });
-};
+@Service()
+export default class {
```

Comment:
Can we name this class JWTService or something like that?

Reply:
Yes, I'll do the same for the S3 service class as well.